src/main/python/systemds/operator/nodes/list.py

```diff
@@ -75,11 +74,6 @@ def pass_python_data_to_prepared_script(self, sds, var_name: str, prepared_scrip
             prepared_script.setMatrix(var_name, numpy_to_matrix_block(
                 sds, self._np_array), True)  # True for reuse
 
-    def code_line(self, var_name: str, unnamed_input_vars: Sequence[str],
-                  named_input_vars: Dict[str, str]) -> str:
-        code_line = super().code_line(var_name, unnamed_input_vars, named_input_vars)
-        return code_line
-
```

Comment:
removed since we call anyways the super class method, so no need to overwrite imo